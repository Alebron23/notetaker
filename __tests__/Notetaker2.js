import React from 'react';
import Notetaker from '../src/components/Notetaker.js';
import renderer from 'react-test-renderer'
import Cookies from 'universal-cookie';


// jest.mock('universal-cookie', () => {
//     console.log('RUN')
//     class FakeCookie {
//         constructor() {
//             console.log('CONSTRUCTOR')
//         }

//         get(key) {
//             console.log('GET')
//             return ['fake', 'fake2']
//         }
//         set(key, val) {
            
//         }
//     }

//     return FakeCookie
// });



it('notaker practice', () => {
    const cookies = new Cookies
    const displayedNotes = ['foo', 'bar', 'baz']
    cookies.set('NOTES', displayedNotes)

    const component = renderer.create(<Notetaker />); 
    expect(component.toJSON()).toMatchSnapshot();
})





