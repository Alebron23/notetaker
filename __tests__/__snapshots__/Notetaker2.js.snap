// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`notaker practice 1`] = `
<div
  className="container"
>
  <h1>
    Note To Self
  </h1>
  <form
    className=""
    onSubmit={[Function]}
  >
    <input
      className="form-control"
      id={undefined}
      name={
        Array [
          "foo",
          "bar",
          "baz",
        ]
      }
      onChange={[Function]}
      placeholder="Enter Note"
      type="text"
      value=""
    />
    <div
      style={
        Object {
          "height": "5px",
          "width": "5px",
        }
      }
    >
       
    </div>
    <button
      className="btn btn-default"
      disabled={false}
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "purple",
          "color": "white",
        }
      }
      type="button"
    >
      Submit 
    </button>
  </form>
  <hr />
  <div
    className="container"
  >
    <h3>
      Notes
    </h3>
    <div
      className="space"
    >
      <div
        className="note"
      >
        <form
          className="form-inline"
          name={0}
          onSubmit={[Function]}
        >
          <input
            className="form-control"
            id={undefined}
            name={0}
            onChange={[Function]}
            type="text"
            value="foo"
          />
          <button
            className="button btn btn-default"
            disabled={false}
            name={0}
            onClick={[Function]}
            type="button"
          >
            X
          </button>
        </form>
      </div>
    </div>
    <div
      className="space"
    >
      <div
        className="note"
      >
        <form
          className="form-inline"
          name={1}
          onSubmit={[Function]}
        >
          <input
            className="form-control"
            id={undefined}
            name={1}
            onChange={[Function]}
            type="text"
            value="bar"
          />
          <button
            className="button btn btn-default"
            disabled={false}
            name={1}
            onClick={[Function]}
            type="button"
          >
            X
          </button>
        </form>
      </div>
    </div>
    <div
      className="space"
    >
      <div
        className="note"
      >
        <form
          className="form-inline"
          name={2}
          onSubmit={[Function]}
        >
          <input
            className="form-control"
            id={undefined}
            name={2}
            onChange={[Function]}
            type="text"
            value="baz"
          />
          <button
            className="button btn btn-default"
            disabled={false}
            name={2}
            onClick={[Function]}
            type="button"
          >
            X
          </button>
        </form>
      </div>
    </div>
    <div
      className="static-modal"
    />
    <div
      className="static-modal"
    />
  </div>
</div>
`;
